### PR TITLE
Add proto message for control plain meta 

### PR DESCRIPTION
**Pull request by Hosshii**
**What this PR does / why we need it**:
Add proto message for control plain meta 

**Which issue(s) this PR fixes**:

rel #1958

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/server/service/webservice/service.proto [hunk 1]
```diff
@@ -499,3 +502,15 @@ message ListEventsResponse {
...
+}
+
+message GetControlPlainMetaResponse {
+    message Version {
+	   string Version = 1;  
+	   string GitCommit = 2;
+	   string BuildDate = 3; 
+    }
+    Version sever = 1;
+}
```
Comment (khanhtc1202) [inline pkg/app/server/service/webservice/service.proto:516]: missing new line at the end of file.

File: pkg/app/server/service/webservice/service.proto [hunk 2]
```diff
@@ -96,6 +96,9 @@ service WebService {
 
     // Events
     rpc ListEvents(ListEventsRequest) returns (ListEventsResponse) {}
+
+    // ControlPlain metadata
+    rpc GetControlPlainMeta(GetControlPlainMetaRequest) returns (GetControlPlainMetaResponse) {}
```
Comment (khanhtc1202) [inline pkg/app/server/service/webservice/service.proto:101]: I guess without implementation for this rpc, the `build` will not pass. Should add implementation for it in `web_api.go` too.

Comment (Hosshii) [reply pkg/app/server/service/webservice/service.proto:101]: Thank you. I'll add it

Comment (khanhtc1202) [inline pkg/app/server/service/webservice/service.proto:101]: typo, the name should be `GetControlPlaneMeta` not `Plain`

**Comment by khanhtc1202** [comment]
> @khanhtc1202 I wonder if there is any way to set the version value while building then use it directly.

Nice idea, we have that information at build time, so I think we could pass it as an environment variable and use `proccess.env.XYZ` to get it and stick to the web console source code 🤔

**Comment by Hosshii** [comment]
I thought the client and server versions were independent, but if they are not, I feel like I can embed them at build time

**Comment by nghialv** [comment]
> I thought the client and server versions were independent, but if they are not, I feel like I can embed them at build time

Thank you, guys. So let's give it a try.

**Pull request closed without merging**